cpp/include/cudf/copying.hpp

```diff
@@ -77,13 +77,15 @@ enum class out_of_bounds_policy : bool {
  * use `DONT_CHECK` when they are certain that the gather_map contains only valid indices for
  * better performance. If `policy` is set to `DONT_CHECK` and there are out-of-bounds indices
  * in the gather map, the behavior is undefined. Defaults to `DONT_CHECK`.
+ * @param stream CUDA stream used for device memory operations and kernel launches.
```

Comment:
Echoing @davidwendt's request for consistency in the removal of periods (I saw that thread earlier but didn't notice it again when scrolling through the PR diff).

Reply:
Fixed.